intg/src/main/java/org/apache/atlas/model/lineage/LineageListRequest.java

```diff
@@ -25,6 +25,15 @@ public class LineageListRequest {
     private Set<String>                     relationAttributes;
     private Boolean                         excludeMeanings;
     private Boolean                         excludeClassifications;
+    private Boolean                         immediateNeighbours=false;
```

Comment:
Let's rename this to "includeImmediateNeighbours". The pattern is similar above - excludeMeanings, excludeClassifications.